.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "tools/mman_win32"]
+	path = tools/mman_win32
+	url = https://github.com/SeoHyungjun/mman-win32.git
+[submodule "tools/iniparser_win32"]
+	path = tools/iniparser_win32
+	url = https://github.com/SeoHyungjun/iniparser.git
```

Comment:
1. .gitmodules is unconditionally activated regardless of OS. So, please get these sources in a Windows build script. (or a script called by Windows build script.)
2. Don't use your personal git repo for this.

Reply:
1. Submodules have been added to conduct unit tests in Windows first. So I tried to make a script or batch file later. 
Let me write a script first. Is there no problem if I make it with python?

2. If I can't use personal git repo, can add a window-resource repo to nnstreamer?

Reply:
1. No. python is an overkill and you are creating another dependency. Use something already have: Windows Shell (batch file) or VS Project Script.

2. Yes.

Reply:
Anyway, you really need to mark a git commit ID for both packages regardless whether you use gitmodules or not.

You may use gitmodules and fix it later or add a note to gitmodules and README that gitmoduels are required for Windows builds only.